Demos/Benchmark/BenchmarkDemo.cpp

```diff
@@ -335,25 +489,28 @@ void PrintParameters(BenchmarkConfig benchmark)
     std::cout << std::endl
               << "This benchmark demo produces timings of a configurable simulation setup." << std::endl
               << "<N> participants exchange <M> messages of <B> bytes per simulation step" << std::endl
-              << "with a fixed period of 1ms and run for <S> seconds (virtual time)." << std::endl
+              << "with a period of <T> ms and run for <S> seconds (virtual time)." << std::endl
               << "This simulation run is repeated <K> times and averages over all runs are calculated." << std::endl
-              << "The demo uses PubSub controllers with the same topic for the message exchange," << std::endl
-              << "so each participant broadcasts the messages to all other participants." << std::endl
+              << "The demo can be run using PubSub, CAN or Ethernet controllers for the message exchange," << std::endl
+              << "where each participant broadcasts the messages to all other participants." << std::endl
               << std::endl
               << "Running simulations with the following parameters:" << std::endl
               << std::endl
-              << std::left << std::setw(38) << "- Number of simulation runs: " << benchmark.numberOfSimulationRuns
+              << std::left << std::setw(39) << "- Communication service type: " << benchmark.service << std::endl
+              << std::left << std::setw(39) << "- Number of simulation runs: " << benchmark.numberOfSimulationRuns
               << std::endl
-              << std::left << std::setw(38) << "- Simulation duration (virtual time): " << benchmark.simulationDuration
+              << std::left << std::setw(39) << "- Simulation step size (virtual time): " << benchmark.simulationStepSize
               << std::endl
-              << std::left << std::setw(38) << "- Number of participants: " << benchmark.numberOfParticipants
+              << std::left << std::setw(39) << "- Simulation duration (virtual time): " << benchmark.simulationDuration
               << std::endl
-              << std::left << std::setw(38) << "- Messages per simulation step (1ms): " << benchmark.messageCount
+              << std::left << std::setw(39) << "- Number of participants: " << benchmark.numberOfParticipants
               << std::endl
-              << std::left << std::setw(38) << "- Message size (bytes): " << benchmark.messageSizeInBytes << std::endl
-              << std::left << std::setw(38) << "- Registry URI: " << benchmark.registryUri << std::endl
-              << std::left << std::setw(38) << "- Configuration: " << benchmark.silKitConfigPath << std::endl
-              << std::left << std::setw(38) << "- CSV output: " << benchmark.writeCsv << std::endl;
+              << std::left << std::setw(39) << "- Messages per simulation step (1ms): " << benchmark.messageCount
```

Comment:
Step size is configurable now, so the "1ms" is no longer valid
```suggestion
              << std::left << std::setw(39) << "- Messages per simulation step: " << benchmark.messageCount
```